cmd/ping.go

```diff
@@ -264,12 +193,105 @@ func fetchAdminInfo(admClnt *madmin.AdminClient) (madmin.InfoMessage, error) {
 			if e == nil {
 				return info, nil
 			}
-
 			timer.Reset(time.Second)
 		}
 	}
 }
 
+func ping(ctx context.Context, cliCtx *cli.Context, anonClient *madmin.AnonymousClient, admInfo madmin.InfoMessage, endPointMap map[string]serverStats, index int) {
+	var endPointStats []EndPointStats
+	var servers []madmin.ServerProperties
+	if cliCtx.Bool("distributed") {
+		servers = admInfo.Servers
+	} else {
+		servers = append(servers, admInfo.Servers[0])
```

Comment:
It is confusing why always the first server, which can be offline by the way. Can't we enhance anonClient.Alive() a little bit to return the status of the node that receives the request if there is no specified servers list ?

Reply:
PR https://github.com/minio/madmin-go/pull/102 raised for the same @vadmeste